entrypoint.sh

```
@@ -0,0 +1,157 @@
+source venv/bin/activate
+echo "=== Debugging Environment Variables ==="
+env  # Print all environment variables
+
+echo "=== Checking Specific Variables ==="
+echo "MYSQL_DATABASE: $MYSQL_DATABASE"
+echo "MYSQL_USER: $MYSQL_USER"
+echo "MYSQL_PASSWORD: $MYSQL_PASSWORD"
+echo "MYSQL_ROOT_PASSWORD: $MYSQL_ROOT_PASSWORD"
+
+echo "=== Starting Application ==="
+exec "$@"
```

Shellcheck (0.10.0)

[warning] Remove "exec " if script should continue after this command.

(SC2093)